stock_location_tray/views/stock_location_views.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_location_form_tray_type" model="ir.ui.view">
```

Comment:
Wee should keep the original XML-ID if possible (`view_location_form`), the namespace being the module now.

Reply:
I used to do that, but it's everywhere different, so I do not know anymore what's better :)
I'll use the original xmlid.